main.go

```diff
@@ -41,14 +42,21 @@ func main() {
 			fmt.Printf("Error: unable to parse repository name %s.\n", entry)
 			os.Exit(1)
 		}
-
+		if v.fork == "true" {
+		// start a new  build
+		_, err = client.BuildFork(owner, name, build.Number)
```

Comment:
build, err :=
and afterwards you should print a message like in the else at the end